evaluation/benchmarks/swe_bench/scripts/eval_infer.sh

```diff
@@ -139,3 +139,38 @@ else
         --max_workers $N_PROCESS \
         --run_id $RUN_ID
 fi
+
+
+
+# Add this to eval_infer.sh after the line with update_output_with_eval.py
+echo "=============================================================="
+echo "Updating localization success statuses"
+echo "=============================================================="
+
+LOC_EVAL_FILEPATH='/shared/nas2/xuehangg/OpenHands/evaluation/benchmarks/swe_bench/memory_saves'
+
+# Get evaluation results
+if [ -f $RESULT_OUTPUT_DIR/report.json ]; then
+    # Extract successful instance IDs from the report.json
+    SUCCESSFUL_INSTANCES=$(jq -r '.instances[] | select(.success == true) | .instance_id' $RESULT_OUTPUT_DIR/report.json)
+    FAILED_INSTANCES=$(jq -r '.instances[] | select(.success == false) | .instance_id' $RESULT_OUTPUT_DIR/report.json)
+    
+    # Update localization success for successful instances
+    for instance_id in $SUCCESSFUL_INSTANCES; do
+        echo "Updating success status for $instance_id: SUCCESS"
+        poetry run python evaluation/benchmarks/swe_bench/memory/update_success.py \
+            "$instance_id" --success --eval-dir $(dirname $LOC_EVAL_FILEPATH)
+    done
+    
+    # Update localization success for failed instances
+    for instance_id in $FAILED_INSTANCES; do
+        echo "Updating success status for $instance_id: FAILURE"
+        poetry run python evaluation/benchmarks/swe_bench/memory/update_success.py \
+            "$instance_id" --eval-dir $(dirname $LOC_EVAL_FILEPATH)
+    done
+    
+    echo "Localization success statuses updated"
+else
+    echo "Report file not found: $RESULT_OUTPUT_DIR/report.json"
+    echo "Cannot update localization success statuses"
+fi
```

Comment:
If localization task is so different, let's create a new folder `swe_bench_localization` with their own scripts, and only keep relevant files there (i.e., import modules/functions from `swe_bench/`)